GitHubA11yFixes.user.js

```diff
@@ -36,70 +54,9 @@ function onDropdownChanged(target) {
 	}
 }
 
-// Used when we need to generate ids for ARIA.
-var idCounter = 0;
 
-function onNodeAdded(target) {
+function doGlobal(target){

```

Comment:
While I'm fine with refactor around the area one is changing, I don't understand how this is related. (I don't understand why it's useful at all, actually; why is site-wide stuff more "separate" than page specific stuff?) The major problem is that this really obscures the actual change you're making here. In addition, there's been some change to this area of the code since you filed this PR (sorry about that), so this won't apply properly. I think this part should be reverted from this PR and filed as a separate PR if you still feel it's necessary.

Reply:
You are correct.